tests/unit/core/test_tracking.js

```diff
@@ -353,4 +357,194 @@ describe(__filename, () => {
       sinon.assert.calledWith(fakeLog.log, 'Do Not Track is enabled');
     });
   });
+
+  describe('Hybrid Content Telemetry filterIdentifier()', () => {
+    it('should return content up to the default max length', () => {
+      expect(filterIdentifier('abcdefghijklmnopqrstuvwxyz').length).toEqual(20);
+    });
+
+    it('should return content up to the supplie max length', () => {
+      expect(
+        filterIdentifier('abcdefghijklmnopqrstuvwxyz', { maxLen: 10 }).length,
```

Comment:
You could also repeat a letter `N` times to be more explicit (I had to count the number of letters). I think `'a'.repeat(20)` should work. This comment also applies to the previous test case.